price-feeder/tests/integration/provider_test.go

```diff
@@ -69,12 +72,12 @@ func (s *IntegrationTestSuite) TestSubscribeCurrencyPairs() {
 
 	time.Sleep(25 * time.Second) // wait for provider to connect and receive some prices
 
-	checkForPrices(s.T(), pvd, currencyPairs)
+	checkForPrices(s.T(), pvd, currencyPairs, "Okx")
```

Comment:
should be "OKX"

Reply:
https://github.com/umee-network/umee/pull/1791/commits/ceeb48c9decee07a966821efeb5285fc8f15afb4